sdk/trace/src/main/java/io/opentelemetry/sdk/trace/export/BatchSpanProcessor.java

```diff
@@ -72,7 +72,7 @@ public static BatchSpanProcessorBuilder builder(SpanExporter spanExporter) {
             scheduleDelayNanos,
             maxExportBatchSize,
             exporterTimeoutNanos,
-            JcTools.newMpscArrayQueue(maxQueueSize));
+            JcTools.newFixedSizeQueue(maxQueueSize));
```

Comment:
I think the `JcTools` helper class might be a bit misnamed now, with this change.

Reply:
The intent is to use JcTools so I think the name is still ok, especially since it only fails in a very small amount of cases so I think it's still a reasonable name

Reply:
> I think the `JcTools` helper class might be a bit misnamed now, with this change.

I personally agree with that, my first PR was doing the try/catch in the `BatchSpanProcessor`, but then I was pointed in another direction.

Reply:
We'll definitely use the queue in at least another class, BatchLogProcessor, so keeping it centralized is good.

@jkwatson Does the thumbs up count as an approval? :)

Reply:
oh yeah..forgot I hadn't approved (I think I was waiting for a passing build). I'm ok with this for now, although the naming is still a bit weird to me.

Reply:
Feel free to send a PR to rename to `ConcurrentQueues` if it sounds better :P